test/write_events.h

```diff
@@ -181,6 +190,51 @@ void write(std::string outfilename) {
     std::cout << "\n collection:  " << "SimCalorimeterHits" <<  " of type " <<  schs.getValueTypeName() << "\n\n"
         << schs << std::endl ;
 
+
+    //===============================================================================
+    // write some TPCHits:
+    int ntpch = 5 ;
+    for(int j=0 ; j<ntpch ; ++j){
+      auto tpch1 = tpchs.create() ;
+      tpch1.setCellID( 0xabadcaffee ) ;
+      tpch1.setTime( j*3. ) ;
```

Comment:
```suggestion
      tpch1.setTime( j*3.f ) ;
```
I think if you set the time here as `float` (instead of as an implicit `double`) you can simplify the reading side of this